exercises/python/encryption-context-complete/setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="Busy Engineer's Document Bucket",
+    version="0.1.0",
```

Comment:
Are we following semver.org advice here? I am not the biggest fan of using 0.x releases, but my comment is not blocking. https://codeblog.jonskeet.uk/2019/10/20/why-i-dont-start-versions-at-0-x-any-more/

Reply:
Few things:
1. This isn't actually going to get packaged or released anywhere, just distributed in source on GitHub for people to hack on and work through. So the versioning is not the usual scope/impact.
1. I'm doing the versioning thing (and `CHANGELOG`!) partially for our own recordkeeping / sanity.
1. I'm taking a swag at using https://www.conventionalcommits.org which implies SemVer.

I read your link though it seems the argument that it's arbitrary itself hinges on an arbitrary distinction... 

>Consistency. With this scheme, never releasing anything with starting with “0.” ... Whereas if you use a major version of 0 as well, just version 1.0.0 is treated specially. It gets pre-releases of 0.1, 0.2, 1.0.0-alpha01, 1.0.0-alpha02, 1.0.0-beta01 etc. I’m fine with things being inconsistent when there’s a good reason for it, but I don’t see any benefit here.

Anyway. I'm not bothered either way, really, and this isn't really the final versioning scheme anyway. I've been meaning to cut an issue to plumb through the Conventional Commits stuff, so I've done that; hopefully that addresses things well enough but let me know if you feel otherwise.

https://github.com/aws-samples/busy-engineers-document-bucket/issues/22